lib/dndstrings-captions.sty

```diff
@@ -195,4 +195,64 @@
 %    \renewcommand\spelldurationname{Duration}
   }
 
+% German captions
+\addto\captionsngerman{%
+    \def\armorclassname{Rüstungsklasse}%
+    \def\hitpointsname{Trefferpunkte}%
+    \def\speedname{Bewegungsrate}%
+    \def\strstatname{Str}%
+    \def\dexstatname{Ges}%
+    \def\constatname{Kon}%
+    \def\intstatname{Int}%
+    \def\wisstatname{Wei}%
+    \def\chastatname{Cha}%
+    \def\skillsname{Fertigkeit}%
+    \def\dimmname{Immunitäten}%
+    \def\dvulname{Anfälligkeiten}%
+    \def\dresname{Wiederstandsschaden}%
+    \def\cimmname{Immunität}
+    \def\savesname{Rettung}%
+    \def\sensesname{Sinne}%
+    \def\languagesname{Sprachen}%
+    \def\challengename{Herausforderung}%
+    \newcommand\xpname{RXP}
+    % Monster attack
+    \renewcommand\ftname{Fuß}
+    \renewcommand\meleeattackname{Nahkampfangriff}
+    \renewcommand\rangedattackname{Fernkampangriff}
+    \renewcommand\meleeorrangedattackname{Nah- oder Fernkampfangriff}
```

Comment:
Because the file uses `\ExplSyntaxOn`, spaces are removed when the token list is typeset. You need to include a tilde (~) where ever there should be a space.

Reply:
Understood.